challenges/classes.js

```diff
@@ -1,11 +1,59 @@
 // Copy and paste your prototype in here and refactor into class syntax.
+// function CuboidMaker (length, width, height) {
+//   this.length = length;
+//   this.width = width;
+//   this.height = height;
+// }
+//
+// CuboidMaker.prototype.volume = function () {
+//   return this.length * this.width * this.height;
+// }
+//
+// CuboidMaker.prototype.surfaceArea = function () {
+//   return 2 * ((this.length * this.width) + (this.length * this.height) + (this.width * this.height));
+// }
 
+class CuboidMaker {
+  constructor(length, width, height) {
```

Comment:
You could pass in `props` to constructor and proceed with `this.length = props.length;` and so forth.